docs/pipeline-components-and-applications/loaders-storage-targets/snowflake-streaming-loader/configuration-reference/_kafka_config.md

```diff
@@ -0,0 +1,28 @@
+<tr>
```

Comment:
For all our common-streams apps, these kinesis/kafka/pubsub configuration inclusions are going to be the same everywhere.  One day when I have more energy, I might split them out to a common re-usable block in the docs site too.